src/validation.h

```diff
@@ -83,6 +84,15 @@ static const int DEFAULT_STOPATHEIGHT = 0;
 static const unsigned int MIN_BLOCKS_TO_KEEP = 288;
 static const signed int DEFAULT_CHECKBLOCKS = 6;
 static const unsigned int DEFAULT_CHECKLEVEL = 3;
+static const std::vector<std::string> CHECKLEVEL_DOC = {
```

Comment:
```suggestion
extern const std::vector<std::string> CHECKLEVEL_DOC;
```

In the header (for expensive types, not integers) this should just be the declaration. The rest should go to the cpp file.

Reply:
Ah ok I should use extern